test_files/samples/element_extra_file_info_2.txt

```diff
@@ -0,0 +1,2 @@
+**/pds:logical_identifier*
```

Comment:
Given the filename, I assumed this would include "extra file" elements like `filepath`.

Reply:
Because `filepath` has to be invoked at the command line, it is generated at the very end of the index file generation. 

I will change the filename to something else.